lib/shopify_app/app_proxy_verification.rb

```diff
@@ -3,7 +3,7 @@ module AppProxyVerification
     extend ActiveSupport::Concern
 
     included do
-      skip_before_action :verify_authenticity_token
+      skip_before_action :verify_authenticity_token, raise: false
```

Comment:
so to confirm:- current shopify_app passed the test using ruby >= v2.3 required by current version of shopify_api ?